lib/node_modules/@stdlib/stats/base/ndarray/sstdev/docs/types/test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/* eslint-disable space-in-parens */
+
+import zeros = require( '@stdlib/ndarray/zeros' );
+import scalar2ndarray = require( '@stdlib/ndarray/from-scalar' );
+import sstdev = require( './index' );
+
+
+// TESTS //
+
+// The function returns a number...
+{
+	const x = zeros( [ 10 ], {
+		'dtype': 'float32'
+	});
+	const correction = scalar2ndarray( 1.0, { 'dtype': 'float32' } );
```

Comment:
```suggestion
	const correction = scalar2ndarray( 1.0, {
		'dtype': 'float32'
	});
```

Be consistent. See L30-32.